src/components/buttonKey.js

```diff
@@ -0,0 +1,29 @@
+function ButtonKey() {
+  return (
+    <div className="calculator">
+      <div className="button-key">
+        <button data="AC" className="btn" type="button">AC</button>
+        <button data="+/-" className="btn" type="button">+/-</button>
+        <button data="%" className="btn" type="button">%</button>
+        <button data="÷" className="btn-operator" type="button">÷</button>
```

Comment:
- [ ] You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray: